lib/beaker-hostgenerator/generator.rb

```diff
@@ -101,25 +97,25 @@ def get_host_roles(node_info)
         roles << role
       end
 
-      return roles
+      roles
     end
 
     private
 
     def generate_host_roles!(host_config, node_info, options)
-      if not options[:disable_default_role]
-        host_config['roles'] = ['agent']
-      else
-        host_config['roles'] = []
-      end
+      host_config['roles'] = if options[:disable_default_role]
+                               []
+                             else
+                               ['agent']
+                             end
```

Comment:
Perhaps make it a ternary instead?
```suggestion
      host_config['roles'] = options[:disable_default_role] ? [] : ['agent']
```